test/unit/account/search_test.rb

```diff
@@ -65,6 +65,22 @@ class Account::SearchTest < ActiveSupport::TestCase
     assert_equal 'fo\/o', Account.search_ids('fo/o').query
   end
 
+  test 'search with query by email' do
+    ThinkingSphinx::Test.run do
+     provider = FactoryBot.create(:provider_account)
+     buyer = FactoryBot.create(:simple_buyer, :provider_account => provider)
+     buyer2 = FactoryBot.create(:simple_buyer, :provider_account => provider)
+     buyer.users << FactoryBot.build(:admin, :account => buyer, :username => "uname1", :email => "foo@bar.co.uk")
+     buyer2.users << FactoryBot.build(:admin, :account => buyer2, :username => "uname2", :email => "foo@bar.com")
```

Comment:
I don't know if we are allowed to use `foo@bar.co.uk` and `foo@bar.com` because they might belong to somebody 👀 I feel that it is safer to use `foo@bar.co.example.com` and `foo@example.org` or anything that ends with `example.*` 😄

Reply:
Another detail: for hashes, plz try to use `foo: 'bar'` instead of `:foo => 'bar'` 😄